time/time.go

```diff
@@ -24,6 +27,9 @@ func (t *Time) MarshalJSON() ([]byte, error) {
 // The time is expected to be in Deis' datetime format.
 func (t *Time) UnmarshalText(data []byte) (err error) {
 	tt, err := time.Parse(DeisDatetimeFormat, string(data))
+	if err != nil {
```

Comment:
We should probably check if the error is specifically a [ParseError](https://golang.org/src/time/format.go?s=21241:21287#L591).